simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/configuration/SimulatorConfiguration.java

```diff
@@ -30,4 +31,27 @@
          */
         private int connectors;
     }
+
+    @Data
+    public static class WebSocketConfiguration {
+        /**
+         * Call timeout in milliseconds
+         */
+        private Long callTimeout;
+
+        /**
+         * Connection timeout in milliseconds
+         */
+        private Long connectTimeout;
```

Comment:
can we have the following for each variable?
```
private static final long DEFAULT_CONNECT_TIMEOUT = 5;
private long connectTimeout = DEFAULT_CONNECT_TIMEOUT;
```

Then we'll be able to see current default values and don't have to check for nulls if they're not overridden.